tt_debug: can't resolve println

I think I'm following the docs for `tt_debug`, but something weird happens.

Code:

```rust
fn main() {
    tt_call! {
        macro = [{ tt_is_ident }]
        input = [{ foo }]
        ~~> tt_debug
    }
}
```

Error:

```
error[E0433]: failed to resolve. Could not find `println` in `$crate`
  --> examples/test_debug.rs:7:5
   |
7  | /     tt_call! {
8  | |         macro = [{ tt_is_ident }]
9  | |         input = [{ foo }]
10 | |         ~~> tt_debug
11 | |     }
   | |_____^ Could not find `println` in `$crate`
```

Tried versions:
- `rustc 1.29.0-nightly (9fd3d7899 2018-07-07)`
- `rustc 1.29.0-nightly (e5f6498d3 2018-07-10)`

Any ideas?

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -778,7 +778,7 @@ macro_rules! private_if_branch {
 /// type = [{ Vec < u8 > }]
 /// rest = [{ , compressed = false }]
 /// ```
-#[macro_export(local_inner_macros)]
+#[macro_export]
 macro_rules! tt_debug {
     {
         $(

